### PR TITLE
Fdp 3016 cv spike

### PR DESCRIPTION
**Pull request by hbt29**

### CODE REVIEW

File: src/templates/cv_template.html [hunk 1]
```diff
@@ -226,72 +261,514 @@
...
+         <div class="heading_10">Lighting Designs</div>
+         <div class="regular_10">
+          <% _.forEach(lightingDesigns, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+       <% if (motionGraphicss.length > 0) { %>
```
Comment (gregburton) [inline src/templates/cv_template.html]: maybe rename "motionGraphics" ? (see also line 453)

File: src/templates/cv_template.html [hunk 2]
```diff
@@ -226,72 +261,514 @@
...
+         <div class="heading_10">Motion Graphics</div>
+         <div class="regular_10">
+          <% _.forEach(motionGraphicss, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+       <% if (multipleTypess.length > 0) { %>
```
Comment (gregburton) [inline src/templates/cv_template.html]: maybe rename "multipleTypes" ? (see also line 464)

**Comment by gregburton** [comment]
@hbt29 sending back to you, just minor stuff

File: src/templates/cv_template.html (hunk 1 above)
Comment (hbt29) [reply src/templates/cv_template.html]: So, these artisticWorks are populated automatically using transform... it pluralizes all kinds of artisticWorks... With the above one, the type itself has 's' at the end so it adds a extra one...might be a fair bit of rework...

File: src/templates/cv_template.html (hunk 2 above)
Comment (hbt29) [reply src/templates/cv_template.html]: So, these artisticWorks are populated automatically using transform... it pluralizes all kinds of artisticWorks... With the above one, the type itself has 's' at the end so it adds a extra one...might be a fair bit of rework...

Comment (gregburton) [reply src/templates/cv_template.html]: Ah, makes sense. I'll approve in that case.

**Comment by nymbyl** [comment]
There is the "pluralize" method I added to the widgets_parser for that purpose. It has stuff like this:
> case "thesis": { return "theses" }

So it would just be another case to add e.g.:
> case "multipleTypes": {return "multipleTypes"} 

That would work, I believe?

**Pull request merged**